kolibri/core/exams/api.py

```diff
@@ -69,6 +69,7 @@ class ExamViewset(ValuesViewset):
         "creator",
         "data_model_version",
         "learners_see_fixed_order",
+        "instant_report_visibility",
```

Comment:
The only thing I can think of here, is that we should probably return `True` for this if the backend value is `null` - that will save the frontend from having to worry about the nullability of the field, and leave it entirely as a backend concern (as we only really introduced it for the purposes of migration).